includes/woopay/class-woopay-session.php

```diff
@@ -633,10 +633,10 @@ private static function get_formatted_custom_message() {
 		$custom_message = WC_Payments::get_gateway()->get_option( 'platform_checkout_custom_message' );
 
 		$replacement_map = [
-			'[terms_of_service_link]' => wc_terms_and_conditions_page_id() ?
+			'[terms]'          => wc_terms_and_conditions_page_id() ?
```

Comment:
How do these changes affect merchants that already saved their settings with the previous `[terms_of_service_link]` and `[privacy_policy_link]` values?

Reply:
Great observation. Added backwards compatibility in [58ba364](https://github.com/Automattic/woocommerce-payments/pull/7601/commits/58ba364cc8f263158ef55e7fc430b1a77f7d9459). 

I also added a mechanism to migrate these shortcodes, as they get updated.